### ISSUE TITLE
Feature Request: Color change when attempting to buy in city without enough gold.

### ISSUE THREAD

**Issue by BlackSmokeDMax** [civfanatics/CQUI_Community-Edition#250]
Any chance you guys could change the color of the gold amount when you *can't* afford to buy something.

*Currently when you can afford to buy something in your city it is a nice bright white, easy to read.

*Currently when you can't buy something it is a dim red color, the contrast between that and the dark color around it makes it very hard to read, at least with a quick glance. If it were say, a bright red instead, that would make it much easier to read. Or any other high contrast color would be nice, but red does seem like an obvious color for "not enough cash."

![20210207103740_1](https://user-images.githubusercontent.com/19895984/107153055-28937b00-6931-11eb-81a1-f01cae0c8268.jpg)

**Comment by wayneb64** [comment]
I second this.

**Comment by the-m4a** [comment]
This is an easy change, I can look into this once I settle out the issue with the Diplomacy Deal screen
(which I think I have done)

Thank you for the picture, too, it makes it really easy to get the context!

**Comment by BlackSmokeDMax** [comment]
> This is an easy change, I can look into this once I settle out the issue with the Diplomacy Deal screen
> (which I think I have done)
> 
> Thank you for the picture, too, it makes it really easy to get the context!

Thanks for this!

And of course, get rid of any existing issues first. I wasn't even hoping you would get to my request soon, was just putting it out there as a long term thing to consider!

**Comment by the-m4a** [comment]
I don't want it to be too bright, but I think this is reasonable?
![image](https://user-images.githubusercontent.com/8787640/107865206-0c07af00-6e19-11eb-897a-92be63e9d8a3.png)

This was a single-line change, at least for that production panel, so it's all good and easy :)

**Comment by the-m4a** [comment]
didn't mean to close this one until I actually get it checked into our code

**Comment by wayneb64** [comment]
Much better, I would go a touch brighter.

**Comment by BlackSmokeDMax** [comment]
That is definitely way better, but I do agree with wayne, just a touch brighter would be better.

**Comment by the-m4a** [comment]
Ok, went a touch brighter - the version on the workshop has the one I have in the picture above, though... I'll push this one to the workshop sometime in the next few days
![image](https://user-images.githubusercontent.com/8787640/107920656-2aa59d00-6f22-11eb-93b2-6c4af275badb.png)

**Comment by the-m4a** [comment]
and by sometime in the next few days, I guess I meant within the next hour.  I figured out why the industry and corp banners weren't showing, so I fixed that and wanted to upload it to the workshop.  The brighter text went with it.
While I was at it, I made it so that if you did not have enough faith to purchase something, it would also show as red.  
Also figured out why the religious units never quite appeared correctly in the unit list!
![image](https://user-images.githubusercontent.com/8787640/107933582-c5f33e00-6f33-11eb-8f8d-08ddabb0ce88.png)
